test/public/vidoe.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { FramelessPostMocks } from '../framelessPostMocks';
```

Comment:
The name of this file is misspelled, can you please rename it to video.spec.ts?

Reply:
Thanks Erin for pointing it out. Will rename it.